doc/connectivity/bluetooth/autopts/autopts-linux.rst

```diff
@@ -338,15 +369,17 @@ At the first run, when Windows asks, enable connection through firewall:
    :align: center
 
 Troubleshooting
-================
+****************
 
-- "After running one test, I need to restart my Windows virtual machine to run another, because of fail verdict from APICOM in PTS logs."
+After running one test, I need to restart my Windows virtual machine to run another, because of fail verdict from APICOM in PTS logs
+====================================================================================================================================
```

Comment:
Not sure if it make a lot of sense semantically to use a sub-title here. IMO it should be a citation but this is a nitpick 😅

Reply:
I like the way that the Arch Wiki does this for troubleshooting (e.g. https://wiki.archlinux.org/title/VirtualBox#Keyboard_and_mouse_are_locked_into_virtual_machine). Not only is it more searchable (headers are usually treated different than paragraph text by search engines), but it also allows linking to specific points in the troubleshooting guide and will give a quick overview in the TOC

I'm in favor of keeping it as a section instead of a quote block

Reply:
I like how it's done there too. But they don't write in first person. Anyway it's not a big deal ^^